README.md

```diff
@@ -0,0 +1,17 @@
+# otm-website-tests
+
+An automated test suite for the OpenTreeMap website.
+
+## Usage
+
+To execute the test suite, first install [Leiningen](http://leiningen.org/), then execute the following command:
+
+```bash
+$ lein midje
```

Comment:
It would be helpful to document that you can use and env var to point the test suite at any host ``OTM_BASE_URL=http:/somehost:8181 lein midje``